src/lib/tecs/TECS.cpp

```diff
@@ -68,9 +68,15 @@ void TECSAirspeedFilter::update(const float dt, const Input &input, const Param
 		return;
 	}
 
+	if (!airspeed_sensor_available) {
+		_airspeed_state.speed = param.equivalent_airspeed_trim;
+		_airspeed_state.speed_rate = 0.f;
+		return;
+	}
+
```

Comment:
Do we really want this? If you have a airspeed sensor failure mid air, this will result in a jump in the airspeed filter. (although i guess it won't be a high jump). I think you did this to get rid of the initialization at 0 mid air, but this could be done as well in the TECS::initialize function as well? I am not strongly against your solution just want to raise this for discussion and you could do it the other way as well.

Reply:
Fair point. Let's do that: 87f73c3714a1667860a61acb522b3d100aaf1150